DjangoCon Europe Support: Provide description of responsibilities (#21)
django/dsf-working-groups: active/dceu.md

> ## Scope of responsibilities

Reviewer: I’m surprised there isn’t a line item about participating in the call for proposal and vetting process, beyond the group being a point of contact. Is that intentional?

Author: I'm not sure what you mean, there are two line items about exactly that:

> - Provide advice and assitance on creating and executing the Call for Proposal process for future conferences
> - Assist the board in vetting incoming proposals and make recommendations for how to approach the decision (the final decision is always made by a board vote)

I deliberately phrased them in a way that the group assists the board, not takes over responsibility for these steps, as my impression wasn't that the board wants to delegate these things entirely, just have a source for knowledgeable input.

Reviewer: My bad, I just missed it :) This is exactly what I was expecting.

The DjangoCon Europe Support working group supports prospective and current organizers in the following ways:

> - Provide a point of contact for people intersted in organizing the conference to ask questions

Reviewer: ```suggestion
- Provide a point of contact for people interested in organizing the conference to ask questions
```

                         
## Future membership

> Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com.

Reviewer: ```suggestion
Membership is open to former organizers of DjangoCon Europe. Questions about membership can be directed to european-organizers-support@djangoproject.com.
```

Membership is self-managed: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.

## Budget

No budget needed at this time.

## Comms

The team internally operates via a Google Group. This is also the primary external point of contact to reach the working group: european-organizers-support@djangoproject.com

> The working group is also present in the #europe channel on DSF slack as a public place of discussion.

Reviewer: This seems ok to me but do keep in mind the DSF Slack is only for the inner circle of people who are currently active with DSF working groups / Django teams / DSF-sponsored initiatives.

Author: Ah, wasn't sure about that. Should we rather remove this line then?

Reviewer: I’m happy either way

> The working group is also present in the #europe channel on DSF slack as a public place of discussion.

Reviewer 2: +1 for either removing it or stating the current modus operandi:

```suggestion
The working group uses the #europe channel on DSF's Slack to coordinate internally, with the DSF board and with other DSF working groups.
```

The team will also set up a collection of data and information, in something like a Google Drive folder. A static informational website might follow at a later date.

## Reporting

> As DjangoCon Europe has a native yearly cycle, the working group will formally report to the board once per year. However, since the working group is not making any decisions and does most of its work by advising the board, the board is expected to be updated on important DjangoCon Europe matters in real-time througout the year.

Reviewer: To me as a board member yearly isn’t frequent enough. My expectation would be monthly or quarterly. I’d rather receive short but frequent reports, as otherwise I could be out of the loop on what the group is up to for a while. A yearly cycle sounds great to me to make a public report of the group’s activities, but for the board I’d like to know about progress on:

1. Our selection process. Both the methodology, and practically where we’re at with different proposals / EoIs.
2. How the group works with this year’s organizers.
3. Our organizational knowledge improvements.

We also use reporting as a way to keep track of group health (see [Shutting down WGs](https://github.com/django/dsf-working-groups#shutting-down-wgs)).

Author: Looking at the months the WG has existed now, there was not a single piece of "action" within the WG (like when vetting the 2025 proposals or giving feedback on the form) where no board member (you, Cagil, Chaim) was deeply involved. I assume that is going to stay that way since

- the board formally receives the proposals
- the board makes the decisions in the selection process

So at least for item 1 of your list, if I would now send a report for this quarter that says "Becky worked with Thibauld to vet the proposals for 2025", that did not appear incredible useful to me.

But if you prefer, we can just change it to a monthly cycle, especially for the "How the group works with this year’s organizers" part it makes sense. Although I assume that, since the 2025 team has prior experience, they will not need the support from us very much, but I'm okay with sending a sometimes-almost-empty email once a month.

Reviewer: Let’s see what others think. From my perspective, if you think there’s not much to report on, a report of "there isn’t much to report on" is completely fine. We can always ask for more info.

> there was not a single piece of "action" within the WG […] where no board member was deeply involved

Board members don’t always report on their activities with the rest of the board so I don’t think that’s relevant?

Reviewer: Just noting without further feedback, I’m happy to move forward with the existing wording :)
